flink-core/src/test/java/org/apache/flink/core/memory/DataInputDeserializerTest.java

```diff
@@ -54,4 +54,14 @@ public void testAvailable() throws Exception {
         }
         Assert.assertEquals(0, dis.available());
     }
+
+    @Test
+    public void testReadWithLenZero() throws IOException {
+        byte[] bytes = new byte[] {};
+        DataInputDeserializer dis = new DataInputDeserializer(bytes, 0, bytes.length);
+        Assert.assertEquals(0, dis.available());
+
+        byte[] bytesForRead = new byte[] {};
+        Assert.assertEquals(0, dis.read(bytes, 0, 0)); // do not throw when read with len 0
```

Comment:
nit: should here be `dis.read(bytesForRead, 0, 0))` ?

Reply:
fixed.